spec.html

```diff
@@ -32156,6 +32156,33 @@ <h1>
         </emu-alg>
       </emu-clause>
 
+      <emu-clause id="sec-time-zone-identifiers">
+        <h1>Time Zone Identifiers</h1>
+
+        <p>
+          Time zones in ECMAScript are represented by <dfn variants="time zone identifier">time zone identifiers</dfn>, which must be Strings composed entirely of characters from the Unicode Basic Latin block.
+          Time zone identifiers are compared using ASCII case-insensitive comparisons.
+          Time zones supported by an ECMAScript implementation may be <dfn variants="available named time zone">available named time zones</dfn>, represented by the [[Identifier]] field of the Time Zone Identifier Records returned by AvailableNamedTimeZoneIdentifiers, or <dfn variants="offset time zone">offset time zones</dfn>, represented by Strings for which IsTimeZoneOffsetString returns *true*.
+        </p>
+        <p>
+          A <dfn variants="primary time zone identifiers">primary time zone identifier</dfn> represents the preferred identifier of an available named time zone, while a <dfn variants="non-primary time zone identifiers">non-primary time zone identifier</dfn> is an alias for the same available named time zone.
+          Each available named time is represented by exactly one primary time zone identifier and zero or more non-primary time zone identifiers.
+          The set composed of all primary and non-primary time zone identifiers that are supported by an ECMAScript implementation are called <dfn variants="available named time zone identifier">available named time zone identifiers</dfn>.
+        </p>
+        <p>
+          At a minimum, ECMAScript implementations must support an available named time zone with the identifier *"UTC"*, which must be primary.
+          If the return value _tz_ of SystemTimeZoneIdentifier is different from *"UTC"*, and IsTimeZoneOffsetString(_tz_) is *false*, then implementations must also support _tz_ as an available named time zone identifier, and it must be primary as well.
+          In addition, implementations may support any number of other available named time zones.
+        </p>
+        <p>
+          Implementations that follow the requirements for time zones as described in the ECMA-402 Internationalization API specification are called <dfn>time zone aware</dfn>.
+          Time zone aware implementations must support available named time zones corresponding to the Zone and Link names of the IANA Time Zone Database (and <b>only</b> such names) and use best available current and historical information about their offsets from UTC and their daylight saving time rules in calculations as specified in the ECMA-402 specification.
```

Comment:
Do we need to say how these names correspond to primary/non-primary time zone identifiers?

Reply:
Sure I can add some text for this.

Reply:
I split this huge sentence in two and sandwiched the primary/non-primary info in the middle. Better?

> Time zone aware implementations must support available named time zones corresponding to the Zone and Link names of the IANA Time Zone Database, and only such names.
> In time zone aware implementations, a primary time zone identifier is a Zone name, and a non-primary time zone identifier is a Link name, respectively, in the IANA Time Zone Database except as specifically overridden by AvailableNamedTimeZoneIdentifiers as specified in the ECMA-402 specification.
> Time zone aware implementations must use the best available current and historical information about their offsets from UTC and their daylight saving time rules in calculations.